### ISSUE TITLE
blessings.Terminal colored output

### ISSUE THREAD

**Issue by slollo** [bpython/bpython#646]
Ubuntu 16.04:

```
% bpython3
bpython version 0.15 on top of Python 3.5.2 /usr/bin/python3
>>> from blessings import Terminal
>>> t = Terminal()
>>> t.red('This is red.')
'\x1b[31mThis is red.\x1b(B\x1b[m'
>>> print(t.red('This is red.'))
This is red.
Traceback (most recent call last):
  File "/usr/bin/bpython3", line 9, in <module>
    load_entry_point('bpython==0.15', 'console_scripts', 'bpython')()
  File "/usr/lib/python3/dist-packages/bpython/curtsies.py", line 190, in main
    exit_value = repl.mainloop()
  File "/usr/lib/python3/dist-packages/bpython/curtsies.py", line 125, in mainloop
    self.process_event_and_paint(e)
  File "/usr/lib/python3/dist-packages/bpython/curtsies.py", line 99, in process_event_and_paint
    array, cursor_pos = self.paint()
  File "/usr/lib/python3/dist-packages/bpython/curtsiesfrontend/repl.py", line 1356, in paint
    self.lines_for_display)
  File "/usr/lib/python3/dist-packages/bpython/curtsiesfrontend/replpainter.py", line 39, in paint_history
    lines.append(fmtstr(line[:columns]))
  File "/usr/lib/python3/dist-packages/curtsies/formatstring.py", line 617, in fmtstr
    string = FmtStr.from_str(string)
  File "/usr/lib/python3/dist-packages/curtsies/formatstring.py", line 156, in from_str
    tokens_and_strings = parse(s)
  File "/usr/lib/python3/dist-packages/curtsies/escseqparse.py", line 27, in parse
    front, token, rest = peel_off_esc_code(rest)
  File "/usr/lib/python3/dist-packages/curtsies/escseqparse.py", line 79, in peel_off_esc_code
    d['numbers'] = [int(x) for x in d['numbers'].split(';')]
  File "/usr/lib/python3/dist-packages/curtsies/escseqparse.py", line 79, in <listcomp>
    d['numbers'] = [int(x) for x in d['numbers'].split(';')]
ValueError: invalid literal for int() with base 10: ''
%
```

**Comment by thomasballinger** [comment]
What version of curtsies do you have installed?

**Comment by slollo** [comment]
python3-curtsies 0.2.6-1 from the repository

**Comment by thomasballinger** [comment]
This should be fixed in more recent versions of Curtsies, dup of https://github.com/bpython/bpython/issues/624

**Comment by thomasballinger** [comment]
I just tried and indeed this example fails for <=0.2.7, it was fixed in f9fe134c377dc6a23b77b7f1505f3c844750af17
